submissions/cynthiaNwakaeme/php-database/cookies.php

```diff
@@ -1,55 +1,69 @@
 <?php
-require_once('connection.php');
 
 echo "<h2> List of Customers at Taries Empire </h2>";
 
-try{
-    /* ACCESSING E-COMMERCE DATABASE AND QUERYING THE DATABASE*/
-    $use_db = "USE taries_empire";
-    $sql = "SELECT * FROM customers";
-    $pdo->exec($use_db);
-    $stmt = $pdo->query($sql);
-    
+class Customer{
+    public function dbQuery(){
+        try{
+            require_once('connection.php');
+
+            /* ACCESSING E-COMMERCE DATABASE AND QUERYING THE DATABASE*/
+            $use_db = "USE taries_empire";
+            $sql = "SELECT * FROM customers";
+            $pdo->exec($use_db);
+            $stmt = $pdo->query($sql);
+            
+            /* GENERATING COOKIES
+            1. result is initalized to get all records from database
+            2. This result is serialize to change the array/object to simple string/text file which is required to create cookies.
+             */
+             $result = $stmt->fetchAll();
+            setcookie("customer", serialize($result), time() + (86400 * 30), "/", "", false); //60sec *60mins *24hours/day => 86400 = 1 day   
+            return($result);

```

Comment:
It is `return $result` not `return($result)`

Reply:
Thank You ma.